build/README.md

```diff
@@ -25,11 +25,9 @@
 * [Apache Thrift 0.9.3](http://archive.apache.org/dist/thrift/0.9.3/)
 
 ## Build command
-Build with different supported versions of Spark, by default using Spark 2.2.1 to build
+Build with different supported versions of Spark, by default using Spark 2.4.4
```

Comment:
default version is spark 2.3.4. will it use spark 2.4 as default version?

Reply:
we can change to spark 2.4 after this PR is merged